[JTC] Add additional parameter to enable configuration of interfaces for following controllers in a chain. (#380)
ros-controls/ros2_controllers: joint_trajectory_controller/doc/userdoc.rst

Details about parameters
^^^^^^^^^^^^^^^^^^^^^^^^

joints (list(string)):

>   Joint names to control and listen to.

Author: ```suggestion
  Specifies names of joints used by the controller.
```

command_joints (list(string)):

>   Joint names to control. This parameters is used if JTC is used in a controller chain where command and state interfaces don't have same names.

Author: ```suggestion
  Specifies names of actuated joints. This parameters is used if JTC is used in a controller chain where command and state interfaces don't have same names.
```
